ios/FluentUI.Demo/FluentUI.Demo/Demos.swift

```diff
@@ -61,7 +63,7 @@ struct Demos {
         DemoDescriptor("BottomCommandingController", BottomCommandingDemoController.self),
         DemoDescriptor("Card", CardViewDemoController.self),
         DemoDescriptor("DateTimePicker", DateTimePickerDemoController.self),
-        DemoDescriptor("NavigationController", NavigationControllerDemoController.self),
+        DemoDescriptor("DrawerController", DrawerDemoController.self),
```

Comment:
Drawer is already in the `fluent2` section, so there must've been a messy merge at some point. Could you remove it from this section?

Reply:
Done.